actions/expirevar.go

```diff
@@ -17,20 +18,35 @@ type expirevarFn struct {
 	key        string
 }
 
-func (a *expirevarFn) Init(r rules.RuleMetadata, data string) error {
-	v, ttl, _ := strings.Cut(data, "=")
-	col, key, ok := strings.Cut(v, ".")
+func (a *expirevarFn) Init(_ rules.RuleMetadata, data string) error {
+	k, ttl, ok := strings.Cut(data, "=")
 	if !ok {
-		return fmt.Errorf("expirevar must contain key and value (syntax expirevar:key=value)")
+		return errors.New("invalid argument for expirevar, requires key and value (syntax expirevar:key=value)")
```

Comment:
Let's extract package variables for fixed errors, `var errRequiresKeyAndvalue` etc

Reply:
Done